portalnet/src/storage.rs

```diff
@@ -766,40 +741,44 @@ impl PortalStorage {
 }
 
 // SQLite Statements
-const CREATE_QUERY: &str = "CREATE TABLE IF NOT EXISTS content_metadata (
+const CREATE_QUERY: &str = "CREATE TABLE IF NOT EXISTS content_data (
                                 content_id_long TEXT PRIMARY KEY,
                                 content_id_short INTEGER NOT NULL,
                                 content_key TEXT NOT NULL,
+                                content_value TEXT NOT NULL,
                                 content_size INTEGER
                             );
-                            CREATE INDEX content_size_idx ON content_metadata(content_size);
-                            CREATE INDEX content_id_short_idx ON content_metadata(content_id_short);
-                            CREATE INDEX content_id_long_idx ON content_metadata(content_id_long);";
+                            CREATE INDEX content_size_idx ON content_data(content_size);
+                            CREATE INDEX content_id_short_idx ON content_data(content_id_short);
+                            CREATE INDEX content_id_long_idx ON content_data(content_id_long);";
 
 const INSERT_QUERY: &str =
-    "INSERT OR IGNORE INTO content_metadata (content_id_long, content_id_short, content_key, content_size)
-                            VALUES (?1, ?2, ?3, ?4)";
+    "INSERT OR IGNORE INTO content_data (content_id_long, content_id_short, content_key, content_value, content_size)
```

Comment:
Before we deploy this change we should add a field to tag which network the content is for. We can probably extract it from the content_key with our new changes, but this might not always be true. Also we will have to use some sql magic if we don't add a tag field, which I would like this avoid.